packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
Well, I think Events™ are the more suitable analogy for Action Types, while the identifiers for action creators should stay closer to Commands™.

`DOCUMENT_INITIALIZING` however is neither of those...

We are pretty inconsistent in this regard... But if we'd want to do things right from here on, I'd suggest to rename these action types to:

* `DOCUMENT_INITIALIZATION_STARTED`
* `DOCUMENT_INITIALIZATION_FINISHED`

And the corresponding action creators to:

* `startDocumentInitialization`
* `finishDocumentInitiazation`

... using the command form however, feels understandibly wrong at the place where the action is dispatched ...